tensorflow_datasets/core/visualization.py

```diff
@@ -85,12 +96,37 @@ def show_examples(ds_info, ds, rows=3, cols=3, plot_scale=3., image_key=None):
 
     image_key = image_keys[0]
 
-  label_keys = [
+  sequence_keys = [
       k for k, feature in ds_info.features.items()
-      if isinstance(feature, features_lib.ClassLabel)
+      if isinstance(feature, features_lib.Sequence)
   ]
 
-  label_key = label_keys[0] if len(label_keys) == 1 else None
+  bbox_keys = []
+  sequence_key = None
+  for key in sequence_keys:
```

Comment:
Add comment to give a high level description of what each sections of your code is doing